addons/l10n_ar/tests/common.py

```diff
@@ -9,7 +9,7 @@
 _logger = logging.getLogger(__name__)
 
 
-@tagged('external_l10n', '-at_install', 'post_install', '-standard', 'external')
+@tagged('-at_install', 'post_install')
```

Comment:
Why?

Reply:
It was just me misunderstanding your previous comment about the tags. 😄